docs/running-notebooks/closing-active-sessions/index.md

```diff
@@ -30,3 +30,29 @@ click on the vertical ellipsis menu on the right-hand-side and select
 "Close Jupyter Notebook".
 
 ![ui-close-session](../../images/ui-close-session.png)
+
+### Auto-closure of inactive sessions
```

Comment:
Went for option 2. In the "opening sessions" page:
- added the auto-closure notice to the first CLI example 
- added a small warning box that mentions the cleanup of inactive session and points to the appropriate paragraph in the "Closing ~active~ sessions" page.

Reply:
Looks nice :+1: Can you please rebase against latest `master`? 

Also, a cosmetic comment: the commit log headline "configuration: customizing notebook auto-closure" might be better said "administration: ..." because it will denote more clearly that this configuration help page is for admins, not for researchers.  (Although we don't systematically use the same prefixes across commits... Something we can agree upon.)